solidity/ecdsa/contracts/libraries/Wallets.sol

```diff
@@ -22,12 +22,19 @@ library Wallets {
         // Uncompressed ECDSA public key stored as X and Y coordinates (32 bytes each).
         bytes32 publicKeyX;
         bytes32 publicKeyY;
+        // This struct doesn't contain `__gap` property as the structure is stored
+        // in a mapping, mappings store values in different slots and they are
+        // not contiguous with other values.
     }
 
     struct Data {
         // Mapping of keccak256 hashes of wallet public keys to wallet details.
         // Hash of public key is considered an unique wallet identifier.
         mapping(bytes32 => Wallet) registry;
+        // Reserved storage space in case we need to add more variables.
+        // See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
+        // slither-disable-next-line unused-state
+        uint256[19] __gap;
```

Comment:
49?

Reply:
I tested it, and what is interesting, switching to 50-element gaps and eliminating them from parameter structs, reduces the contract size.

Now vs altered:
```
 |  WalletRegistry               ·     24.357  │
 |  WalletRegistry               ·     24.196  │
```

Reply:
e0f9ea4fb0d90a13b88e6de5038e5e75fcb51e38